src/pages/home/report/PureReportActionItem.tsx

```diff
@@ -1011,7 +1011,7 @@ function PureReportActionItem({
                             <KYCWall
                                 onSuccessfulKYC={() => Navigation.navigate(ROUTES.ENABLE_PAYMENTS)}
                                 enablePaymentsRoute={ROUTES.ENABLE_PAYMENTS}
-                                addBankAccountRoute={ROUTES.BANK_ACCOUNT_PERSONAL}
+                                chatReport={report}
```

Comment:
We are replacing a static route here. I think we should match this with the chatReportID passed here.
```suggestion
                                chatReport={targetReport}
```

Reply:
hmm good point, I have to dig a little bit since I haven't worked on this PR originally

Reply:
@c3024 I couldn't figure out the proper condition to incorporate this path into `getBankAccountRoute`. Instead, I brought back the `addBankAccountRoute` param as optional, so if it's defined, it will use it in first place. What do you think about this?